src/component.cpp

```diff
@@ -37,8 +37,8 @@ struct Component::ComponentImpl
 
 std::vector<VariablePtr>::iterator Component::ComponentImpl::findVariable(const std::string &name)
 {
-    return std::find_if(mVariables.begin(), mVariables.end(),
-                        [=](const VariablePtr& v) -> bool {return v->getName() == name;});
+    return std::find_if (mVariables.begin(), mVariables.end(),
```

Comment:
Just for information (i.e. don't make it delay the merge!): ```std::find_if()``` is a function, not a an ```if``` statement as such, so I wouldn't expect a space to be 'needed.